src/calculateMaximumColumnWidthIndex.ts

```diff
@@ -7,10 +7,6 @@ import type {
  * Produces an array of values that describe the largest value length (width) in every column.
  */
 export default (rows: Row[]): number[] => {
-  if (!rows[0]) {
```

Comment:
Isn't this code run on the _data_, not the _config_?

Reply:
Yes. On data. But it is checked in validateTableData.ts.